sdks/python/apache_beam/runners/direct/clock.py

```diff
@@ -15,36 +15,35 @@
 # limitations under the License.
 #
 
-"""Clock implementations for real time processing and testing."""
+"""Clock implementations for real time processing and testing.
 
+For internal use only. No backwards compatibility guarantees.
+"""
 from __future__ import absolute_import
 
 import time
 
 
 class Clock(object):
-  """For internal use only; no backwards-compatibility guarantees."""
+  def current_time(self):
+    raise NotImplementedError()
 
-  def time(self):
-    """Returns the number of milliseconds since epoch."""
-    return int(time.time() * 1000)
+  def advance_time(self):
+    raise NotImplementedError()
 
 
-class MockClock(Clock):
-  """For internal use only; no backwards-compatibility guarantees.
+class RealClock(object):
+  def current_time(self):
+    return time.time()
 
-  Mock clock implementation for testing."""
 
-  def __init__(self, now_in_ms):
-    self._now_in_ms = now_in_ms
+class TestClock(object):
+  """Clock used for Testing"""
```

Comment:
<!--new_thread; commit:517fae9dcb749a59322ea40c7de0eb90ebf5efac; resolved:0-->
"testing".

Please document the unit used in `current_time` and `advance_by`.